libcontainer/intelrdt/intelrdt.go

```diff
@@ -67,7 +74,21 @@ import (
  * |       |-- delay_linear
  * |       |-- min_bandwidth
  * |       |-- num_closids
- * |-- ...
+ * |-- mon_data
+ *     |-- mon_L3_00
+ *     	   |-- llc_occupancy
+ *     	   |-- mbm_local_bytes
+ *         |-- mbm_total_bytes
+ *     |-- ...
```

Comment:
How about removing these lines? For runc container use case, we don't touch the mon_data directory under root.